xarray/core/dataset.py

```diff
@@ -743,47 +743,59 @@ def copy(self, deep=False, data=None):
         Shallow copy versus deep copy
 
         >>> da = xr.DataArray(np.random.randn(2, 3))
```

Comment:
Tip for next time: to make a reproducible docstring example, use `np.random.RandomState(0).randn(2, 2)` instead. It's a little longer, but RandomState objects are almost always a better design choice than the global functions.

Reply:
ok sounds good. I tend to use `np.arange(size).reshape(shape)`  but I decided it was too long for this one.